pkg/api/cluster/schema.go

```diff
@@ -0,0 +1,78 @@
+package cluster
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/rancher/apiserver/pkg/store/empty"
+	"github.com/rancher/apiserver/pkg/types"
+	"github.com/rancher/steve/pkg/server"
+
+	"github.com/harvester/harvester/pkg/config"
+)
+
+const (
+	deviceCapacity   = "deviceCapacity"
+	localClusterName = "local"
+)
+
+type Cluster struct {
+	Name string
+}
+
+type Store struct {
+	types.Store
+}
+
+func (f *Store) ByID(_ *types.APIRequest, _ *types.APISchema, id string) (types.APIObject, error) {
+	if id != localClusterName {
+		return types.APIObject{}, fmt.Errorf("only supported cluster id is local")
+	}
+	return types.APIObject{
+		Type: "cluster",
+		ID:   "local",
+		Object: Cluster{
+			Name: "local",
+		},
+	}, nil
+}
+
+func (f *Store) List(_ *types.APIRequest, _ *types.APISchema) (types.APIObjectList, error) {
+	return types.APIObjectList{
+		Objects: []types.APIObject{
+			{
+				Type: "cluster",
+				ID:   "local",
+				Object: Cluster{
+					Name: "local",
+				},
+			},
+		},
+	}, nil
+}
```

Comment:
we should keep schema clear, we can put this store into another file called `store.go` like other one does.

Reply:
sounds good..